### PR TITLE
Issue 382 guidance and examples

### PR DESCRIPTION
**Pull request by mslarae13**
see https://github.com/microbiomedata/nmdc-schema/issues/382 

Updated 4 terms.

### CODE REVIEW

**Review by turbomam — CHANGES_REQUESTED** [review]

File: src/schema/nmdc.yaml [hunk 1]
```diff
@@ -524,7 +524,33 @@ classes:
...
+        examples:
+          - value: 1871-10-10
+          - value: 1871-10-01 to 1871-10-31
+        todos:
+          - is "to" acceptable? Is there a better way to request that be written?
+        annotations:
+          expected_value:
+            tag: expected_value
+            value: date string
+        range: string
```
[the remark this answers is not in the document]
Comment (mslarae13) [reply src/schema/nmdc.yaml:539]: I agree we should implement stricter validation rules. Should we make tickets for each slot that needs better validation rules? Or just a general ticket?

File: src/schema/portal/emsl.yaml [hunk 2]
```diff
@@ -88,11 +94,15 @@ slots:
     recommended: true
   technical_reps:
     name: technical_reps
-    description: If sending multiple technical replicates of the same sample, indicate
-      how many replicates are being sent
+    description: If sending technical replicates of the same sample, indicate the replicate number.
```
[the remark this answers is not in the document]
Comment (mslarae13) [reply src/schema/portal/emsl.yaml:97]: I think 'ordinal identifier (within those replicates) of the sample represent on this row'
So, if someone is sending 3 50mL tubes full of the same soil they would put a 3 here. It's just so at EMSL when we have 1 line, but received 3 tubes we know why.

File: src/schema/portal/mixs_inspired.yaml [hunk 3]
```diff
@@ -78,79 +78,105 @@ slots:
     title: experimental factor- other
     from_schema: https://example.com/nmdc_dh
     see_also:
-    - MIxS:experimental_factor|additional_info
+      - MIxS:experimental_factor|additional_info
     rank: 7
     string_serialization: '{text}'
     slot_group: MIxS Inspired
     recommended: true
+    comments:
```
[the remark this answers is not in the document]
Comment (mslarae13) [reply src/schema/portal/mixs_inspired.yaml:86]: That makes sense! I will add this comment to the open issue and implement it for the next PR.

File: src/schema/nmdc.yaml [hunk 4]
```diff
@@ -669,90 +666,88 @@ classes:
...
+#        #multivalued: true
         range: string
-        examples:
-          - value: 75%
-          - value: 75 %
-          - value: 0.75 g water/g dry soil
-          - value: 75% water holding capacity
-          - value: 1.1 g fresh weight/ dry weight
-          - value: 10% water filled pore space
+#        examples:
```
Comment (mslarae13) [inline src/schema/nmdc.yaml:684]: @turbomam  need some help here. This failed upon 'make all'. What do I need to do here to fix this?

**Comment by mslarae13** [comment]
I'd like this PR to be merged before I make any changes.
All additional updates require some conversation

File: src/schema/nmdc.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (mslarae13) [reply src/schema/nmdc.yaml:538]: @turbomam  thoughts on my response and can we clear this to merge?

File: src/schema/nmdc.yaml (hunk 4 above)
Comment (turbomam) [reply src/schema/nmdc.yaml:685]: works for me. maybe there was something wrong with the indentation. I can show you how to automatically fix some formatting errors like that.

**Pull request merged**